gazebo_state_plugins/src/GazeboMapPublisher.cpp

```diff
@@ -34,6 +34,20 @@
 
 using gazebo::GazeboMapPublisher;
 
+#if GAZEBO_MAJOR_VERSION >= 8
+namespace gz_math = ignition::math;
+#define gz_math_Vector3d ignition::math::Vector3d
```

Comment:
same with `typedef` instead of `#define` which I commented on for the other files